calc/pkg/package.json

```diff
@@ -13,7 +13,6 @@
   "files": [
     "calc_bg.wasm",
     "calc.js",
-    "calc_bg.js",
```

Comment:
Why?

Reply:
That was auto-fixed by yarn or my ide. That file just does not exist.